### ISSUE TITLE
Bikeshed errors

### ISSUE THREAD

**Issue by annevk** [whatwg/mimesniff#15]
```
WARNING: The following <var>s were only used once in the document:
  'samplerate'
  'doctype size'
If these are not typos, please add an ignore='' attribute to the <var>.
LINK ERROR: No 'dfn' refs found for 'whitespace character'.
LINK ERROR: No 'dfn' refs found for 'plugin context'.
LINK ERROR: No 'dfn' refs found for 'style context'.
LINK ERROR: No 'dfn' refs found for 'script context'.
```
@padenot I believe the first two have been introduced by you and it does look like those algorithms have some kind of mistake.

Not really sure how to resolve the remainder or what was intended there. @GPHemsley?

**Comment by GPHemsley** [comment]
You can likely rewrite 'whitespace character' to use Infra. When I wrote this, there wasn't a centralized place where the difference between bytes/characters/code points were defined.

The context terms were extension points for what I took to be likely scenarios at the time. It's possible these have been superseded by Fetch or something else. See https://wiki.whatwg.org/wiki/Contexts for more info.

**Comment by annevk** [comment]
What's remaining here is algorithms using a variable only once. I have some vague plans to go through all algorithms and write tests for them, so hopefully they can be corrected then. However, I wouldn't mind someone else stepping in and doing that work as I don't know when I'll get to it.
